client/src/Components/PreNavBar.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
-import { AppBar } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
 
 //Logo
 //Sign In
-//Sign Up User
+//Sign Up Userimport React from 'react';
```

Comment:
This looks like maybe it just accidentally ended up here so probably delete the import React from 'react' from this comment.